0-bootstrap/variables.tf

```diff
@@ -88,6 +88,17 @@ variable "bucket_tfstate_kms_force_destroy" {
   default     = false
 }
 
+variable "vpc" {
+  description = "The type of VPC to attach the project to. Possible options are base or restricted."
```

Comment:
nit

```suggestion
  description = "The type of VPC to attach the project to. Possible options are none, base, or restricted."
```

Reply:
as "none" would be indicative of no network, and the only possible values ​​are "base" and "restricted", do you think that if we include none it would be necessary to put an indicative in the description, for example, "none" is used to indicate that there is no network created, or would it not be necessary?

Reply:
@mariammartins - Just a suggestion. :) From the description only (without looking at the variable default or condition) a user might assume `vpc` MUST be one of `base` or `restricted`.  But likely a low risk.